File: app/src/main/java/com/example/cnccodegenerator/code_generator/component_types/PathPoint.java
package com.example.cnccodegenerator.code_generator.component_types;

public class PathPoint {
    public int x;
    public int y;
    public boolean isFillet;
    public PathPoint(){
        x=0;
        y=0;
        isFillet = false;
    }
    public PathPoint(int x, int y, boolean isFillet){
        this.x = x;
        this.y = y;
        this.isFillet = isFillet;
    }

}

File: app/src/main/java/com/example/cnccodegenerator/code_generator/component_types/PathPoint.kt
package com.example.cnccodegenerator.code_generator.component_types

class PathPoint {
    var x: Float
    var y: Float
    var isFillet: Boolean

    constructor() {
        x = 0f
        y = 0f
        isFillet = false
    }

    constructor(x: Float, y: Float, isFillet: Boolean = false) {
        this.x = x
        this.y = y
        this.isFillet = isFillet
    }
}
